sqlitecluster/SQLite.cpp

```diff
@@ -827,16 +870,32 @@ int SQLite::_sqliteAuthorizerCallback(void* pUserData, int actionCode, const cha
                                       const char* detail3, const char* detail4)
 {
     SQLite* db = static_cast<SQLite*>(pUserData);
-    return db->_authorize(actionCode, detail1, detail2);
+    return db->_authorize(actionCode, detail1, detail2, detail3, detail4);
 }
 
-int SQLite::_authorize(int actionCode, const char* table, const char* column) {
+int SQLite::_authorize(int actionCode, const char* detail1, const char* detail2, const char* detail3, const char* detail4) {
```

Comment:
can we rename these params to something more explicit?

Reply:
Hah, I just renamed them to *less* explicit because it turns out "table" and "column" don't always contain the table or column name - for instance, for a function, the function name is in `detail2` (we didn't use this before).

Reply:
Damn, yeah I figured :(